plugins/modules/cohesity_plugin.py

```diff
@@ -220,19 +210,23 @@ def download_datastore_plugin(module):
 
 
 def install_plugin(module, filename):
-
+    """"""
     script_dir = module.params.get("script_dir")
     cmd = "rpm -ivh %s -prefix %s" % (filename, script_dir)
     rc, stdout, stderr = module.run_command(cmd)
     # => Any return code other than 0 is considered a failure.
+    if rc:
+        return(False, "Partially installed the Cohesity Plugin")
     return (True, "Successfully Installed the Cohesity plugin")
 
 
 def uninstall_plugin(module):
     try:
-        cmd = "rpm remove cohesity-postgres-connector -y"
+        cmd = "rpm remove " + COHESITY_POSTGRES_CONNECTOR + " -y"
         rc, stdout, stderr = module.run_command(cmd)
         # => Any return code other than 0 is considered a failure.
+        if rc:
+            return(False, "Failed to uninstall the Cohesity plugin")
         return (True, "Successfully Uninstalled the Cohesity plugin")
     except Exception as error:
         raise__cohesity_exception__handler(error, module)
```

Comment:
If the grep fails, then error should be , couldn't find the postgres-connector to uninstall ? be as specific as possible for multiple error exit points